src/contents/en-gb/doc_android.md

```diff
@@ -0,0 +1,255 @@
+- **An Android Device / Emulator:** To test/run the application.
+
+Note: We only support devices with armeabi-v7a/x86_64 support and Android API 24 and above.
+
+## Setup the basic skeleton
+
+The basic setup of the application is available in a git repository. Let's get started by cloning this repository
+
+```
+git clone https://github.com/maidsafe/safe-getting-started-android
+cd safe-getting-started-android
+git checkout boilerplate
+```
+
+Now import this project into your Android Studio IDE. The required dependencies will be installed. Please give appropriate permissions when required.
+
+## Adding the `safe_app_android` library
+
+There are two packages available to develop applications for the SAFE Network. `safe-app-android` for the alpha-2 / local network and `safe-app-android-dev` for mock routing. For easier switching between mock and non-mock versions, we have made use of the [build variants](https://developer.android.com/studio/build/build-variants) feature of Android Studio. We have defined mock and nonMock variants and have added the required dependency as follows in the [`app/build.gradle`](https://github.com/maidsafe/safe-getting-started-android/blob/master/app/build.gradle#L32) file.
+
+```
+android {
+    ...
+    flavorDimensions "app"
+    productFlavors {
+        mock {
+            dimension "app"
+        }
+        nonMock {
+            dimension "app"
+        }
+    }
+
+}
+
+dependencies {
+    repositories {
+        maven {
+            url "https://oss.jfrog.org/artifactory/oss-snapshot-local/"
+        }
+
+    }
+    mockImplementation 'net.maidsafe:safe-app-android-dev:0.1.0-SNAPSHOT'
+    nonMockImplementation 'net.maidsafe:safe-app-android:0.1.0-SNAPSHOT'
+    ...
+}
+```
+
+The API documentation for the safe-app-android library is available at [docs.maidsafe.net/safe_app_java](https://docs.maidsafe.net/safe_app_java)
+
+## Authenticating the app
+
+A SAFE application needs to get an authorization from the user before being able to connect to the network, this is achieved by sending an authorization request to the Authenticator. The `AuthReq` is created and encoded in the `generateAuthURL()` function in the `SafeTodoService` class. Add the following logic that returns the auth URI as a string.
+
+
+To create an `AuthReq` the following is created:
+
+- An instance of `App` which will hold all the basic information about the application.
+- An `AppExchangeInfo` instance which holds all application info that is shared with the Authenticator.
+- An array of `ContainerPermissions` which specifies the permissions for various containers. Here we do not need access to any of the containers(except the App's container), hence we pass an empty array.
+
+```
+App app = new App("net.maidsafe.sample", "Safe TODO Java", "Maidsafe.net", "0.1.0");
+```
+_The app variable is globally available in the `SafeTodoService` class._
+```
+private static final String AUTH_URL_PREFIX = "safe-auth://";
```

Comment:
This line is actually already in the code (L28) so the user doesn't need to add again